lib/embed/embed.dart

```diff
@@ -56,9 +56,10 @@ class PlaygroundMobile {
   PaperIconButton _resetButton;
   PaperTabs _tabs;
 
-  Gist backupGist;
+  Map _lastRun;
```

Comment:
Can we type this more specifically? Create an enum of the sources and have this as a Map<Enum, String>?